### DISCUSSION TITLE
New model: vicuna-13b-GPTQ-4bit-128g (ShareGPT finetuned from LLaMa with 90% of ChatGPT's quality)

### DISCUSSION THREAD

**Discussion by wywywywy** [oobabooga/text-generation-webui#752] [General]
This just dropped. Vicuna quantized to 4bit.

> We introduce Vicuna-13B, an open-source chatbot trained by fine-tuning LLaMA on user-shared conversations collected from ShareGPT. Preliminary evaluation using GPT-4 as a judge shows Vicuna-13B achieves more than 90%* quality of OpenAI ChatGPT and Google Bard while outperforming other models like LLaMA and Stanford Alpaca in more than 90%* of cases. 

https://huggingface.co/anon8231489123/vicuna-13b-GPTQ-4bit-128g

![image](https://user-images.githubusercontent.com/19997747/229650956-9dd31201-71c6-4040-9c8f-b3d5d7bc27e5.png)

For llama.cpp see here: https://lmsysvicuna.miraheze.org/wiki/How_to_use_Vicuna#Use_with_llama.cpp%3A

Reddit discussion: https://www.reddit.com/r/MachineLearning/comments/12ay0vt/p_the_weights_neccessary_to_construct_vicuna_a/

**Comment by Ada123-a** [comment]
From everything ive seen its worse than gpt4-x-alpaca-13B.

**Reply by ye7iaserag** [reply]
I came here to ask for this, thank you!

**Reply by Wladastic** [reply]
I can confirm, it tends to start replacing you after 3-4 messages and runs continoously without you haha

**Reply by OWKenobi** [reply]
Hi Wladastic, it stops doing that when you used it in "Instruct Mode".

**Comment by BarfingLemurs** [comment]
I don't think they pruned out the filter chats properly.
(would rather have it hallucinate than say "As a language model..")
![image](https://user-images.githubusercontent.com/128182951/229652275-e7a78682-0edd-41ce-ad4f-5468827c6fa6.png)

![image](https://user-images.githubusercontent.com/128182951/229652376-a1dc0a71-785f-4b43-a919-8dc28ce90dca.png)

**Comment by St33lMouse** [comment]
I find this sort of sermonizing deeply annoying. I can decide what I think is right and wrong for myself, thank you very much. Since there is no actual safety layer, just a trained habit, can you just tell it not to sermonize or refuse to answer a question?

**Reply by BadisG** [reply]
That's why gpt4-x-alpaca is the best model, it treats us like adults lmao

**Reply by nthverse** [reply]
Agreed, it reduces the amount of use cases by polluting the output. If you are storytelling having "As an AI MODEL" inserted in your output, breaks everything.

**Comment by Recruiter2** [comment]
could oobabooga work with vicuna-13B-1.1-GPTQ-4bit?
I am struggling to make this version work with oobabooga but it works with llama.cpp 
maybe I should try something else? vicuna-13B-1.1-GPTQ-4bit but not for the llama.cpp

**Reply by Recruiter2** [reply]
couldn't create issue too much steps 'but did you check if that question exist'

**Reply by 3rdeyemedia** [reply]
did you ever get it to work dude?

**Reply by Recruiter2** [reply]
ye it works fine (on cpp of course)
all the repos (oga n llama) been updated since tho so digging commits is required
docs is poor...
I moved on since...

why u necropost like this tho?

ps
13b gptq (thus gpu) worked few prompts 8gb gpu w  one of the (vicuna 13b 4bit models)
u need to close everything +use linux
but now it's all old tech there are better options that take advantage of all of ur hw available juice...
